client/ayon_core/tools/publisher/control.py

```diff
@@ -675,6 +797,18 @@ def has_errors(self):
 
         return bool(self._error_items)
 
+    @property
+    def has_blocking_errors(self):
+        """At least one blocking error was added."""
+
+        return bool(self._blocking_errors)
```

Comment:
This is a bit heavier to compute, but avoids having to pass that much data for this single purpose.
```suggestion
        return any(error.is_blocking for error in self._error_items)
```

Reply:
I used this in `PublishValidationErrorsReport` instead of `PublishValidationErrors`. This change was possible because the data was only meant to be passed to `PublishValidationErrorsReport` and wasn't necessary in `PublishValidationErrors`, which has been reverted to its original state from the upstream repository)